### DISCUSSION TITLE
Lesson 8 Error: No such file or directory

### DISCUSSION THREAD

**Discussion by BowtiedHarpyEagle** [Cyfrin/foundry-full-course-cu#1393] [Q&A]
Hi, 
I'm going through lesson 8 html-fund me. I'm trying to deploy the foundry-fundme to anvil, as it is described in the "introduction to window.ethereum" video. I have my foundry-fund-me-f23 folder cloned from github. But when I try to deploy it on anvil, I get the error file not found. 

I see the folders forge-std and chainlink-brownie-contracts, but they are empty. I think that's because they were excluded in .gitignore files. 

How can I re-install those lib scripts? I don't want to go through all of lesson 7 again. 

I tried `forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit` and `forge install forge-std` those commands also give me errors as well.

**Comment by BowtiedHarpyEagle** [comment]
I figured out how to add those libraries, by removing the corresponding folders first.

**Reply by Joel-eth** [reply]
Please show me how you did that
